scvi/model/_scanvi.py

```diff
@@ -200,7 +200,7 @@ def from_scvi_model(
         scanvi_model = cls(
             adata, unlabeled_category, **non_kwargs, **kwargs, **scanvi_kwargs
         )
-        scvi_state_dict = scvi_model.module.state_dict()
+        scvi_state_dict = scvi_model.module.state_dict().copy()
```

Comment:
I think we might need deepcopy here. We should add a test to check that it works -- pick any tensor inside the state dict and see if it's not equal

Reply:
So, within Pytorch's `load_state_dict` function (the call we make right below this one), they do a simple copy:
```
       # copy state_dict so _load_from_state_dict can modify it
        metadata = getattr(state_dict, '_metadata', None)
        state_dict = state_dict.copy()
```
Could you explain why we would need a deepcopy in our case?

Sure, I will add a test.

Reply:
Talked offline. We don't think we need a `copy()` or `deepcopy()` as PyTorch does this internally.
I added a test though.